bower.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "adapt-contrib-hotgraphic",
-  "version": "2.0.7",
-  "framework": "^2.0.0",
+  "version": "2.0.8",
```

Comment:
The extent of the changes to this and the other components that are being changed in this way maybe merits a bump to 2.1.0?